packages/lesswrong/server/collections/ultraFeedEvents/mutations.ts

```diff
@@ -15,12 +16,22 @@ export const graphqlUltraFeedEventTypeDefs = gql`
     data: CreateUltraFeedEventDataInput!
   }
 
+  input UpdateUltraFeedEventDataInput ${
+    getUpdatableGraphQLFields(schema)
+  }
+
+  input UpdateUltraFeedEventInput {
+    selector: SelectorInput!
```

Comment:
I think maybe we should move away from using `SelectorInput` by default for new update mutations, since there isn't a good reason to keep supporting both `_id` and `documentId`.  Need to think about it